BFE_RShiny/oasisui/R/exposurevalidation_map_module.R

```diff
@@ -671,23 +704,32 @@ exposurevalidationmap <- function(input,
            .between_min_max(circle_bounds_3[, 2], lat[x])) ||
           (.between_min_max(circle_bounds_4[, 1], long[x]) &&
            .between_min_max(circle_bounds_4[, 2], lat[x]))) {
-        uploaded_locs_input[x]
+        uploaded_locs_input[x,]
       }
-    }))
+    })
   }
 
   # show TIV table for pins under circle
   .showPinsInfo <- function(radius, lat_click, long_click, ratio) {
     info_circles <- .DrawnCircles(radius, lat_click, long_click, ratio)
+    if (length(info_circles$tiv_list) > 100) {
+      sorted_tivs <- sort(info_circles$tiv_list, decreasing = TRUE)[1:100]
+      sorted_entries <- unlist(lapply(sorted_tivs, function(x) {which(info_circles$tiv_list == x)}))
```

Comment:
use vectorization here instead of the `lapply`.
I think in the line above what you actually want is `order()` rather than `sort()`, then you should be able to use the resulting indices directly.